server/utils.go

```diff
@@ -210,3 +212,44 @@ func validateLabels(labels map[string]string) error {
 	}
 	return nil
 }
+
+func mergeEnvs(imageConfig *v1.Image, kubeEnvs []*pb.KeyValue) []string {
+	envs := []string{}
+	if kubeEnvs == nil && imageConfig != nil {
+		envs = imageConfig.Config.Env
+	} else {
+		for _, item := range kubeEnvs {
+			if item.GetKey() == "" {
+				continue
+			}
+			envs = append(envs, item.GetKey()+"="+item.GetValue())
+		}
+		if imageConfig != nil {
+			for _, imageEnv := range imageConfig.Config.Env {
+				var found bool
+				parts := strings.SplitN(imageEnv, "=", 2)
+				if len(parts) != 2 {
+					continue
```

Comment:
Are any of these errors?  Is this expected?

Reply:
nope, just ignore like docker does